lib/nexpose/common.rb

```diff
@@ -117,14 +113,12 @@ def as_xml
       xml.attributes['start'] = @start
       xml.attributes['maxDuration'] = @max_duration if @max_duration
       xml.attributes['template'] = @scan_template_id
-      xml.attributes['force'] = @force if @force
       xml
     end
 
     def from_hash(hash)
       schedule = AdHocSchedule.new(hash[:start], hash[:scan_template_id])
       schedule.max_duration = hash[:max_duration] if hash[:max_duration]
-      schedule.force = hash[:force] if hash[:force]
```

Comment:
This require a cucumber test to make sure schedule are saved without force and default value is maintain